__tests__/app.test.js

```diff
@@ -296,6 +287,23 @@ describe("GET /api/articles", () => {
           expect(res.body).toEqual({ message: "Requested URL not found" });
         });
     });
+    test("404: Responds with 404 error no content found when topic 'paper' is specified in query and it is valid, but no articles of that topic exists", () => {
```

Comment:
This should be a 200 - but with an empty array returned. It's not technically an error, because nothing is technically going wrong, we will just have nothing to return to them. You'll need to think about how you're going to tell the difference between an array that's empty because the topic doesn't exist, and an array that's empty because the topic does exist but it has no articles.